lib/wickedbase.pm

```diff
@@ -234,6 +234,31 @@ sub get_test_result {
     }
 }
 
+sub get_wicked_logs {
+    my ($self, $prefix) = @_;
+    my $dir_name = $self->{name} . '_' . $prefix;
+    my $logs_dir = "/tmp/$dir_name";
+    record_info('Logs', "Collecting logs in $logs_dir");
+    script_run("mkdir -p $logs_dir");
+    script_run("date +'%Y-%m-%d %T.%6N' > $logs_dir/date");
+    script_run("journalctl -b -o short-precise|tail -n +2|grep wicked > $logs_dir/journalctl.log");
```

Comment:
Why `| grep wicked` ?

Reply:
ah didn't notice this , agree we need to collect whole log no one knows what will be actually needed by developer

Reply:
aren't we intested only about wicked messages/logs?

Reply:
no , because wicked problems may be introduced by some Kernel oops for example or anything else what you can't imagine

Reply:
ok